Applications/Spire/Source/Ui/SideInputWidget.cpp

```diff
@@ -0,0 +1,23 @@
+#include "Spire/Ui/SideInputWidget.hpp"
+#include <QHBoxLayout>
+#include "Spire/Ui/StaticDropDownMenu.hpp"
+
+using namespace boost::signals2;
+using namespace Nexus;
+using namespace Spire;
+
+SideInputWidget::SideInputWidget(QWidget* parent)
+    : QWidget(parent) {
+  auto menu = new StaticDropDownMenu({QVariant::fromValue<Side>(Side::ASK),
+        QVariant::fromValue<Side>(Side::BID)}, this);
```

Comment:
There is no need to explicitly instantiate function templates in situations where they can be deduced based on the function arguments.

Reply:
In this case I might have to because the type is being deduced as: Nexus::SideEnumMembers::Type, which isn't registered as a QVariant type.

Reply:
Good, good.